services/web-app/pages/mdx/index.mdx

```diff
@@ -141,7 +141,7 @@ platea dictumst.
   </ArticleCard>
 </Column>
 <Column colMd={4} colLg={4} noGutterMdLeft>
-    <ArticleCard
+    <
```

Comment:
what's happening here? 😅  this is breaking the mdx page for me 
<img width="1212" alt="image" src="https://user-images.githubusercontent.com/40550942/173887633-94d9683e-a2e5-494c-8e28-337b214d3809.png">